src/console-services.yml

```diff
@@ -159,22 +303,90 @@
   description: 'Lambda - Run Code without Thinking about Servers'
   url: 'https://console.aws.amazon.com/lambda/home'
 
-- command: machinelearning
-  description: 'Machine Learning - Build Smart Applications Quickly and Easily'
-  url: 'https://console.aws.amazon.com/machinelearning/home'
+- command: lex
+  description: 'Lex - Build Voice and Text Chatbots'
+  url: 'https://console.aws.amazon.com/lex/home'
+
+- command: licensemanager
+  description: 'License Manager - Set rules to manage, discover, and report third-party license usage proactively'
+  url: 'https://console.aws.amazon.com/license-manager/home'
+
+- command: lightsail
+  description: 'Lightsail - Launch and Manage Virtual Private Servers'
+  url: 'https://lightsail.aws.amazon.com/ls/webapp/home'
+
+- command: macie
+  description: 'Macie - Amazon Macie classifies and secures your business-critical content.'
+  url: 'https://macie.aws.amazon.com/'
+
+- command: managedservices
+  description: 'Managed Services - IT operations management for AWS'
+  url: 'https://console.aws.amazon.com/managedservices/home'
+
+- command: marketplace
+  description: 'Marketplace Subscriptions - Digital catalog where you can find, buy, and deploy software'
+  url: 'https://console.aws.amazon.com/marketplace/home'
+
+- command: mediaconnect
+  description: 'MediaConnect - Reliable, secure, and flexible transport for live video'
+  url: 'https://console.aws.amazon.com/mediaconnect/home'
+
+- command: mediaconvert
+  description: 'MediaConvert - Convert file-based content for broadcast and multiscreen delivery'
+  url: 'https://console.aws.amazon.com/mediaconvert/home'
+
+- command: medialive
+  description: 'MediaLive - Convert video inputs into live outputs for broadcast and streaming delivery'
+  url: 'https://console.aws.amazon.com/medialive/home'
 
-- command: mobileanalytics
-  description: 'Mobile Analytics - Collect, View and Export App Analytics'
-  url: 'https://console.aws.amazon.com/mobileanalytics/home'
+- command: mediapackage
+  description: 'MediaPackage - Deliver video to many devices using just-in-time format conversion'
+  url: 'https://console.aws.amazon.com/mediapackage/home'
 
-- command: mobilehub
-  description: 'Mobile Hub - Build, Test, and Monitor Mobile Apps'
-  url: 'https://console.aws.amazon.com/mobilehub/home'
```

Comment:
`mobilehub` seems deprecated, but still a valid service. we should keep this for now

Reply:
OK. Adding back